packages/number-input/src/Select/Select.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+
+import { Select } from '.';
+
+const selectProps = {
+  onChange: jest.fn(),
+  unit: 'day',
+  unitOptions: [
+    {
+      displayName: 'day',
+      value: 'day',
+    },
+    {
+      displayName: 'month',
+      value: 'month',
+    },
+    {
+      displayName: 'year',
+      value: 'year',
+    },
+  ],
+};
+
+function renderSelect(props = {}) {
+  // @ts-ignore - missing props
```

Comment:
should be able to type renderSelect props as `Partial<SelectProps>` to avoid ts-error